From e942dfa8f6f09c1d6f4501794e9616fc4a903e8f Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?St=C3=A9phane=20Marchesin?= <marcheu@chromium.org>
Date: Mon, 8 Jul 2013 20:35:05 -0700
Subject: [PATCH] i915: Force X-tiling

Y-tiling is faster in general, but for Chrome OS we favour upload
speed (which is better with X-tiling).
---
 src/gallium/drivers/i915/i915_resource_texture.c | 5 +----
 1 file changed, 1 insertion(+), 4 deletions(-)

diff --git a/src/gallium/drivers/i915/i915_resource_texture.c b/src/gallium/drivers/i915/i915_resource_texture.c
index 3d61794..0f2a874 100644
--- a/src/gallium/drivers/i915/i915_resource_texture.c
+++ b/src/gallium/drivers/i915/i915_resource_texture.c
@@ -184,10 +184,7 @@ i915_texture_tiling(struct i915_screen *is, struct i915_texture *tex)
    if (util_format_is_s3tc(tex->b.b.format))
       return I915_TILE_X;
 
-   if (is->debug.use_blitter)
-      return I915_TILE_X;
-   else
-      return I915_TILE_Y;
+   return I915_TILE_X;
 }
 
 
-- 
1.8.3

